src/main/resources/templates/welcome.html

```diff
@@ -0,0 +1,81 @@
+<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:th="http://www.thymeleaf.org"
+      layout:decorate="~{index.html}">
+<body>
+
+<div class="container-fluid text-sm p-5 bg-light" layout:fragment="content">
+    <div class="display-5">Welcome</div>
+    <p class="lead">Welcome to OWASP WrongSecrets. With this app, we hope you will re-evaluate your secrets
+        management
+        strategy</p>
+    <hr class="my-4">
+    <p>For each of the challenges below: try to find the secret! Enter it in the `Answer to solution` box and
+        score
+        points!
+        Note that some challenges require this app to run on additional infrastructure (see in the links
+        below).</p>
+
+    <div class="row">
```

Comment:
here it looks a bit odd: the row class div should start outside of the fluid container.

Reply:
fixed